inc/class-wpseo-shortlinker.php

```diff
@@ -41,11 +44,7 @@ public function build_shortlink( $url ) {
 	 * @return string The final URL.
 	 */
 	public static function get( $url ) {
-		$version = WPSEO_VERSION;
-
-		$version = apply_filters( 'wpseo_shortlink_version', $version );
```

Comment:
This filter isn't deprecated?

Reply:
We never talked about it anywhere, so.... no? :)